jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -32,7 +34,7 @@ class Shutdown(object):
 
     def __init__(self):
         rospy.loginfo('Start shutdown node.')
-
+        self.client_jp = SoundClient(sound_action='/robotsound_jp', blocking=True)
```

Comment:
Without this action, an error will occur, so it is better to avoid it.

Reply:
I set `blocking=False`(default) due to the reason in the second reply.
Now `SoundClient` uses publisher instead of ActionClient.  
If we want to wait for speaking to finish, ActionClient is better?